packages/gateway/src/durable-objects/cids.js

```diff
@@ -0,0 +1,43 @@
+import { normalizeCid } from '../utils/cid.js'
+
+/**
+ * @typedef {Object} CidUpdateRequest
+ * @property {string} cid
+ * @property {string[]} urls gateway URLs
+ */
+
+/**
+ * Durable Object for keeping CID state.
+ */
+export class Cids1 {
```

Comment:
Please can we rename `Cids1` to be more descriptive?

Reply:
Renaming for CidsTracker, but not super satisfied about the naming. Any suggestions?

Reply:
Me either but...it'll do for now...